src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/CollectionManager.java

```diff
@@ -1,49 +1,102 @@
+import xyz.holocons.mc.holoitemsrevamp.collection.gen1.YozoraMel;
 import xyz.holocons.mc.holoitemsrevamp.collection.gen2.Gen2Collection;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen2.MinatoAqua;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen2.MurasakiShion;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen2.NakiriAyame;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen2.OozoraSubaru;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen2.YuzukiChoco;
 import xyz.holocons.mc.holoitemsrevamp.collection.gen3.Gen3Collection;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen3.HoushouMarine;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen3.ShiranuiFlare;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen3.ShiroganeNoel;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen3.UruhaRushia;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen3.UsadaPekora;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen4.AmaneKanata;
 import xyz.holocons.mc.holoitemsrevamp.collection.gen4.Gen4Collection;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen4.HimemoriLuna;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen4.KiryuCoco;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen4.TokoyamiTowa;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen4.TsunomakiWatame;
 import xyz.holocons.mc.holoitemsrevamp.collection.gen5.Gen5Collection;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen5.MomosuzuNene;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen5.OmaruPolka;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen5.ShishiroBotan;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen5.YukihanaLamy;
 import xyz.holocons.mc.holoitemsrevamp.collection.gen6.Gen6Collection;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen6.HakuiKoyori;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen6.KazamaIroha;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen6.LaplusDarknesss;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen6.SakamataChloe;
+import xyz.holocons.mc.holoitemsrevamp.collection.gen6.TakaneLui;
+import xyz.holocons.mc.holoitemsrevamp.collection.id1.AiraniIofifteen;
+import xyz.holocons.mc.holoitemsrevamp.collection.id1.AyundaRisu;
 import xyz.holocons.mc.holoitemsrevamp.collection.id1.ID1Collection;
+import xyz.holocons.mc.holoitemsrevamp.collection.id1.MoonaHoshinova;
+import xyz.holocons.mc.holoitemsrevamp.collection.id2.AnyaMelfissa;
 import xyz.holocons.mc.holoitemsrevamp.collection.id2.ID2Collection;
+import xyz.holocons.mc.holoitemsrevamp.collection.id2.KureijiOllie;
+import xyz.holocons.mc.holoitemsrevamp.collection.id2.PavoliaReine;
+import xyz.holocons.mc.holoitemsrevamp.collection.misc.Achan;
 import xyz.holocons.mc.holoitemsrevamp.collection.misc.MiscCollection;
-
-import java.util.List;
-import java.util.Map;
-import java.util.stream.Collectors;
+// import xyz.holocons.mc.holoitemsrevamp.collection.stars1.Arurandeisu;
```

Comment:
Why is HoloStars singled out here?

Reply:
The current idolCollections doesn't have any of the holostars collections in either. Should I add those now?

Reply:
Yeah, I forgot about that then.